src/main/java/com/pusher/rest/data/TriggerResult.java

```diff
@@ -0,0 +1,70 @@
+package com.pusher.rest.data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import com.google.gson.Gson;
+
+import com.pusher.rest.data.Result.Status;
+
+public class TriggerResult {
+    static class Data {
+        public final Map<String, String> event_ids;
```

Comment:
I don't think it matters that much since this class' existence is specifically for marshalling. But I changed it in e8f35b1